installation/windows.md

```diff
@@ -85,4 +85,107 @@ There is currently no automatic update script for Windows. To update manually, d
 
 ## Starting openHAB as a service
 
-(work in progress...)
+To install the openHAB process as a service in Windows in order to be able to launch it automatically and run it in the background, follow these steps:
+
+1. Complete the [prerequisites](#prerequisites) and regular [installation](#installation) steps, including the package selection
+2. Issue the following 2 commands in your openHAB console:
+```
+feature:install service-wrapper
+wrapper:install --name "openHAB2" --display "openHAB2" --description "openHAB 2 Service"
+```
+![Wrapper Install_Windows](images/Wrapper_Install_Windows.jpg)
+
+3. Shutdown the currently running OH2 instance and modify the file `C:\openHAB2\userdata\etc\openHAB2-wrapper.conf` and using a text editor, replace all existing content with:
```

Comment:
"Replace all existing" sounds a bit dangerous in the long run. As a user I would expect the file to already be almost correct in its default settings. Future changes and additions should be merged into the file rather than into this documentation article. Instead of "replace all" I'd prefer something like "Adapt the entry OPENHAB_HOME" and so on. Wdyt?